config/locales/en.yml

```diff
@@ -1556,6 +1556,25 @@ applyNext:
       title: New application
       body: You can create a new funding application for %s here
       cta: Start new application
+  dashboardNew:
+    latest:
+      title: Latest application
+    all:
+      title: All applications
```

Comment:
Very minor changes but the dashboard links are capitalized 'Latest Application' and 'All Applications'.

I'm guessing the Welsh translation is already on the way ?

Reply:
We always sentence case the titles (Earl possibly missed this), Welsh translation will only come once we've finalised the screens.